src/pkg/token/token.go

```diff
@@ -65,7 +66,8 @@ func Parse(opt *Options, rawToken string, claims jwt.Claims) (*Token, error) {
 	if err != nil {
 		return nil, err
 	}
-	token, err := jwt.ParseWithClaims(rawToken, claims, func(token *jwt.Token) (interface{}, error) {
+	var parser = jwt.NewParser(jwt.WithLeeway(time.Duration(60) * time.Second))
+	token, err := parser.ParseWithClaims(rawToken, claims, func(token *jwt.Token) (interface{}, error) {
```

Comment:
Hi @an-toine ,

seems like we could use jwt.ParseWithClaims(...) here per https://github.com/golang-jwt/jwt/blob/b27c88965d976075194dc8125f1f9c60bd8cf162/parser.go#L55

Reply:
Should be addressed by 899634c

Reply:
Seems like there's a shortcut  for NewParser().ParseWithClaims() that we could use directly as `jwt.ParseWithClaims()` with parse options
https://github.com/golang-jwt/jwt/blob/b27c88965d976075194dc8125f1f9c60bd8cf162/parser.go#L236C6-L236C21
Anyway, to claim a new parser with options separately may be more readable.